wiki/Accessibility-A11y-Guide.md

```diff
@@ -55,6 +56,8 @@ By default tablet emulators do not contain **Play Store** app and therefore you
 * Clickable items
 * Text and image contrast
 
+[How to use Accessibility Scanner? - Video Tutorial](https://youtu.be/LF5AgGI5H3A)
```

Comment:
Can you move this to the end of the paragraph at line 33. Something like: Here is a video tutorial on how to set up and use the accessibility scanner

Reply:
@adhiamboperes line 33 means at the end of this [section](https://github.com/oppia/oppia-android/blob/b1b3efd6c51ee4e1b898a3678909e7d8fe69a687/wiki/Accessibility-A11y-Guide.md?plain=1#L38)?

Reply:
This [one](https://github.com/oppia/oppia-android/blob/b1b3efd6c51ee4e1b898a3678909e7d8fe69a687/wiki/Accessibility-A11y-Guide.md?plain=1#L33).

Reply:
Done.